public-event-list.md

```diff
@@ -53,8 +53,10 @@ class EventInstance
     public Boolean earlyAccess = false; // True if early access to the EI is being granted as a Membership benefit
     public String seatingType; // "General Admission" or "Pick Your Own Seats"
     public String purchaseUrl; // URL to the direct purchase page for this EI
-    public Boolean soldOut = true; // false unless all TAs are sold out
+    public Boolean soldOut = true; // False unless all TAs are sold out
```

Comment:
These comments are actually in the Apex class and I usually just copy/paste the whole set of inner classes into this doc so can you also make sure these comments make their way back into the Apex file?